projects/packages/connection/legacy/class-jetpack-xmlrpc-server.php

```diff
@@ -98,14 +89,15 @@ public function xmlrpc_methods( $core_methods ) {
 		}
 
 		/**
-		 * Filters the XML-RPC methods available to Jetpack for requests signed only with a blog token.
+		 * Filters the XML-RPC methods available to Jetpack for requests signed both with a blog token or a user token.
 		 *
 		 * @since 3.0.0
 		 *
-		 * @param array $jetpack_methods XML-RPC methods available to the Jetpack Server.
-		 * @param array $core_methods    Available core XML-RPC methods.
+		 * @param array         $jetpack_methods XML-RPC methods available to the Jetpack Server.
+		 * @param array         $core_methods    Available core XML-RPC methods.
+		 * @param \WP_User|bool $user            Information about a given WordPress user. False if authenticated with blog token.
```

Comment:
Do we document in which version a parameter was added to a filter?

Reply:
Yes, thanks for the reminder. Done in https://github.com/Automattic/jetpack/pull/18952/commits/21418d1e5f212492038290abfc17c8a7989ea123